data/mods/ssb/statuses.js

```diff
@@ -955,12 +955,6 @@ let BattleStatuses = {
 		noCopy: true,
 		onStart(pokemon) {
 			this.add(`c|+Lost Seso|<3 ( ͡° ͜ʖ ͡°) DANCE WITH ME ( ͡° ͜ʖ ͡°) <3`);
-			if (pokemon.illusion) return;
-			this.effectData.danceMultiplier = 0;
-		},
-		onAfterMove(pokemon, target, move) {
-			if (pokemon.illusion) return;
-			if (move.flags.dance) this.effectData.danceMultiplier++;
```

Comment:
What about this though?

Reply:
oh yeah. That was code that was never used since Seso got a different move